bin/update_broken_link.sh

```diff
@@ -7,15 +7,16 @@ wget -O /tmp/od-do-canada.jl.gz http://open.canada.ca/static/od-do-canada.jl.gz
 
 rm -rf /tmp/od_linkcheker2.db
 TMPDIR="/home/odatsrv/tmp"
-(time python3 /home/odatsrv/bin/link_check.py --file=/tmp/od-do-canada.jl.gz ) > "${TMPDIR}/link_check.log" 2>&1
+export LD_PRELOAD=/lib64/libgcc_s.so.1
+(time /srv/python/bin/python3 /home/odatsrv/bin/link_check.py --file=/tmp/od-do-canada.jl.gz ) > "${TMPDIR}/link_check.log" 2>&1
 sleep 30
-(time python3 /home/odatsrv/bin/link_check.py --file=/tmp/od-do-canada.jl.gz --quick ) > "${TMPDIR}/link_check.log2" 2>&1
+(time /srv/python/bin/python3 /home/odatsrv/bin/link_check.py --file=/tmp/od-do-canada.jl.gz --quick ) > "${TMPDIR}/link_check.log2" 2>&1
```

Comment:
some timeout urls will be ok if we re-visit them. I am not sure if it is due to our network or theirs.